presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergMetadata.java

```diff
@@ -393,7 +397,8 @@ public ConnectorOutputTableHandle beginCreateTable(ConnectorSession session, Con
         HiveIdentity identity = new HiveIdentity(session);
         String targetPath = getTableLocation(tableMetadata.getProperties());
         if (targetPath == null) {
-            targetPath = getTableDefaultLocation(database, hdfsContext, hdfsEnvironment, schemaName, tableName).toString();
+            String uniqueTableName = useUniqueTableLocation ? String.format("%s_%s", tableName, UUID.randomUUID()) : tableName;
```

Comment:
Should we remove the dashes? Compare

* `my_table-1ec297aa2e8511eb9dfe7bf8d2aea93a`
* `my_table_1ec297aa-2e85-11eb-9dfe-7bf8d2aea93a`

Reply:
yes, without dashes looks better. Removed dashes and moved generation of unique table name to separate private method